src/org/spdx/htmltemplates/LicenseHTMLFile.java

```diff
@@ -72,23 +75,23 @@ public String getSite() {
 		}
 
 		public boolean getIsValid() {
-			return UrlHelper.urlValidator(url);
+			return Valid.urlValidator(url);
 		}
 
 		public boolean getIsLive() {
-			return UrlHelper.urlLinkExists(url);
+			return Live.urlLinkExists(url);
```

Comment:
In building the mustachMap that drives this, add fields for this based on fetching the CrossRef and parsing it.